stdlib/asm/crypto/stark/constants.masm

```diff
@@ -112,7 +116,30 @@ const.TMP6=4294913276
 const.TMP7=4294913280
```

Comment:
Why do we name these `TMP*`? Couldn't we always use more descriptive names?

Reply:
These are basically memory addresses that can be used to store intermediate results needed during the execution of some procedure.

Reply:
I guess my point was more: how much bloat would it create to create a new variable/memory location for each intermediate result? In the same of reading things like `TRACE_LEN` instead of `TMPX`

Reply:
I think quite a lot, for example we sometimes just need to store the hash of some leaf in a Merkle tree so that we can compare against in say a few lines of assembly code, or we might have a partial result of a computation that we need to store temporally to liberate the stack ...